airflow/contrib/example_dags/example_qubole_operator.py

```diff
@@ -0,0 +1,182 @@
+from airflow import DAG
+from airflow.operators import DummyOperator, PythonOperator, BranchPythonOperator
+from airflow.contrib.operators import QuboleOperator
+from datetime import datetime, timedelta
+import filecmp
+import random
+
+seven_days_ago = datetime.combine(datetime.today() - timedelta(7),
+                                  datetime.min.time())
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': seven_days_ago,
+    'email': ['airflow@airflow.com'],
+    'email_on_failure': False,
+    'email_on_retry': False
+}
+
+dag = DAG('example_qubole_operator', default_args=default_args)
+
+def compare_result(ds, **kwargs):
```

Comment:
As I explained above that task instances wont be available for other worker processes, hence we've to use xcom (or something persistent storage) to save cmd_ids. Not sure if its possible in airflow to pickle task runs and use them later.